src/components/pages/jobs/Job.tsx

```diff
@@ -0,0 +1,186 @@
+import React, { useEffect, useContext } from 'react';
+import { useParams } from 'react-router-dom';
+import './job.scss';
+import { JobItem } from 'models/apiModels';
+import Icon from 'components/atoms/icons/icon';
+import { getJob, deleteJob } from 'api/jobs';
+import { baseUrl } from 'constants/apiConstants';
+import { AuthenticateContext, Roles } from 'contexts/authProvider';
+import Modal from 'components/molecules/modal/Modal';
+import { useMobileScreen } from 'hooks/useMobileScreen';
+import { useHistory } from 'react-router-dom';
+import { useToast } from 'hooks/useToast';
+import Button from 'components/atoms/button/Button';
+
+interface IValidJob {
+  jobData: JobItem | undefined;
+}
+
+// return 404 regardless of reasons for not being able to getEventById
```

Comment:
get event by id?

Reply:
Sorry, some reuse of code repercussions..